### PR TITLE
Less expressy

### PR DESCRIPTION
**Pull request by freeall**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,23 +3,31 @@
...
+      return;
     }
 
-    if (body.type === 'url_verification') return res.send(body.challenge);
+    if (body.type === 'url_verification') return res.end(body.challenge);
     if (body.type === 'event_callback') {
       onSlackEvent(body, function (err) {
-        if (err) return res.status(400).send('Please retry');
-        res.send();
+        if (!err) return res.end();
```
Comment (joshuakarjala) [inline index.js]: `res.end('ok')`

File: index.js [hunk 2]
```diff
@@ -3,23 +3,31 @@
...
+        res.statusCode = 400;
+        res.end('Please retry');
```
Comment (joshuakarjala) [inline index.js]: `res.end('please retry');`

**Comment by freeall** [comment]
@joshuakarjala done

**Comment by freeall** [comment]
@joshuakarjala cool, will you release new version?

**Pull request merged**